doc/script_commands.txt

```diff
@@ -11412,3 +11412,37 @@ Example:
     autoloot(10000); // 100.00%
 
 ---------------------------------------
+
+*setdialogalign(<align>);
+
+Set vertical or horizontal align in NPC dialog.
+Valid aligns:
+- horizontal align:
+  DIALOG_ALIGN_LEFT
+  DIALOG_ALIGN_CENTER
+  DIALOG_ALIGN_RIGHT
```

Comment:
@aleos89 @Atemo do we want HALIGN and VALIGN here? or is it okay like this?

Reply:
And then decided? HALING/VALING or do I leave it as is?